src/components/GameField/GameField.css

```diff
@@ -0,0 +1,33 @@
+.game-board {
+  margin: auto;
+  width: 600px;
+  height: 600px;
+  box-sizing: content-box;
```

Comment:
Please use box-sizing: border-box unless absolutely have to use content-box; also, add global styles to override default box-sizing value on all elements